### DISCUSSION TITLE
name of engine

### DISCUSSION THREAD

**Discussion by EthanMG228** [JordanSantiagoYT/FNF-JS-Engine#605] [Q&A]
Soooo i played bambi golden apple and in the menu it has like Golden apple engine. but in reality its kade engine. How to make that? do i need source? like i want to change menu name from JS Engine to Ether Engine. (i'm on lua) i dont know how to explain this more.

**Comment by moxie-coder** [comment]
it *can* be recreated in lua, I'll make a script in a moment

**Reply by EthanMG228** [reply]
oh thanks

**Comment by JordanSantiagoYT** [comment]
assets/data/windowTitleBase.txt

**Reply by EthanMG228** [reply]
Not a window name. Like you go in the menu and in the down left of the screen you see JS Engine. And in the songs too. How to replace it with Abandoned Engine? (lua)

**Reply by JordanSantiagoYT** [reply]
setTextString('EngineWatermark', (text))

**Reply by EthanMG228** [reply]
where i need to paste this script? im dumb and i dunno how to code. where?

**Reply by JordanSantiagoYT** [reply]
> where i need to paste this script? im dumb and i dunno how to code. where?

any lua script, in onCreatePost
